go/batch-submitter/metrics/metrics.go

```diff
@@ -0,0 +1,79 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type Metrics struct {
+	// ETHBalance tracks the amount of ETH in the submitter's account.
+	ETHBalance prometheus.Gauge
+
+	// BatchSizeInBytes tracks the size of batch submission transactions.
+	BatchSizeInBytes prometheus.Histogram
+
+	// NumTxPerBatch tracks the number of L2 transactions in each batch
+	// submission.
+	NumTxPerBatch prometheus.Histogram
+
+	// SubmissionTimestamp tracks the time at which each batch was confirmed.
+	SubmissionTimestamp prometheus.Histogram
```

Comment:
I'm not sure what this metric is meant to show. Each submission has a different timestamp, right? I feel like this is a log event but not really useful as a metric.

Reply:
this was pulled from the existing metrics, I can remove it if we don't currently use it!

Reply:
Ok, if you can't think of a good use case (I can't), then please remove ⚡

Reply:
awesome, it is removed in the current diff 👍